data_science/scripts/asg_utils.py

```diff
@@ -0,0 +1,62 @@
+# -*- encoding: utf-8
+
+import sys
+
+
+DATA_SCIENCE_ASG_TAGS = {
+    'data_science': 'true',
```

Comment:
For our purposes this is `'name': 'jupyter'`

Reply:
Fixed.